softgroup/basis/common-dao/common-dao-api/src/main/java/com/softgroup/common/dao/api/entities/BaseEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.softgroup.common.dao.api.entities;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 09.03.17.
+ */
+
+@Entity
```

Comment:
@Dominys but without this annotation hybernate cannot create any table for descendants entities
![without_annotation](https://cloud.githubusercontent.com/assets/16840046/23835236/665ef8d6-076c-11e7-9f00-a367a4f7cc39.png)